test/integration/004_simple_archive_test/test-archives-bq/archive.sql

```diff
@@ -9,6 +9,6 @@
             updated_at='updated_at',
         )
     }}
-    select * from `{{database}}`.`{{schema}}`.seed
+    select * from `{{target.database}}`.`{{schema}}`.seed
```

Comment:
why `target.database` and not `database`? Or alternatively, why `schema` and not `target.schema`?

Reply:
I changed the archive parsing code to set `database` and `schema` from the `target_database` and `target_schema` configs. This meant that `{{database}}` would interpolate to the target database for the archive, which can be different than the database where the seed lives. Working on a test that exercises this, will remove the extraneous changes though - most of the `target.database` additions i made here aren't actually necessary